docs/cli-reference.md

```diff
@@ -1,5 +1,5 @@
 ---
-slug: cli-usage
+slug: cli-reference
```

Comment:
What’s the reasoning to change the slug? I didn’t see a discussion about that in the PR conversation.

Note there is also this history: https://github.com/returntocorp/semgrep-docs/pull/107#discussion_r575481763

Reply:
I am just trying to be consistent - the file is called cli-usage, but the first header and left navigation entry is CLI reference. @pabloest I changed all the links leading to this file as well.

Reply:
It just seems weird when the main title is CLI reference and the URL states cli-usage.

Reply:
Sounds great. Thanks!